lib/pbench/server/api/resources/users_api.py

```diff
@@ -114,13 +114,7 @@ def post(self):
             self.logger.info(
```

Comment:
Not that you have changed this here, but the logging and return should be outside the `try` block, or in an `else` clause.

Reply:
I removed the logging statement since it was not really needed and a little misleading